style.css

```diff
@@ -0,0 +1,185 @@
+header{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-height: 100rem;
+    margin-top: 0;
+    background-color: rgb(18, 9, 1);
+    margin: auto;
+    max-width: 100%;
+}
+
+.text-area{
+    position: absolute;
+    color: rgb(231, 214, 206);
+    font-family: 'SF Pro Text',-apple-system,BlinkMacSystemFont,Roboto,'Segoe UI',Helvetica,Arial,sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol';    align-content: center;
+    max-width: 30rem;
+    font-weight: 100;
+    top: 40%;
+    left: 8%;   
+}
```

Comment:
we need to use VSCode prettier

Reply:
Yeh, we start using prettier from week 4, i didn't know about it existing before))